examples/compiled/repeat_histogram.vg.json

```diff
@@ -20,10 +20,6 @@
                     "as": "Horsepower"
                 },
                 {
-                    "type": "filter",
```

Comment:
Actually it may be that the bin handles it okay, but linear scale isn't very smart about it too.